src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ResourceSetType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
+{
+    /// <summary>
+    /// Enum to determine the type of Resource Set
+    /// </summary>
+    internal enum ResourceSetType
```

Comment:
>ResourceSetType [](http://example.com/codeflow?start=18&length=15)

since there are only (ever likely to be) two values, should we just use a boolean isDeltaResourceSet property?